runelite-client/src/main/java/net/runelite/client/game/ItemVariationMapping.java

```diff
@@ -75,4 +86,15 @@ public static int map(int itemId)
 	{
 		return MAPPINGS.getOrDefault(itemId, itemId);
 	}
+
+	/**
+	 * Get item ids for provided variation item id.
+	 *
+	 * @param itemId the item id
+	 * @return the item ids
+	 */
+	public static Collection<Integer> getVariations(int itemId)
+	{
+		return INVERTED_MAPPINGS.asMap().getOrDefault(Math.abs(itemId), Collections.singletonList(itemId));
```

Comment:
Are we ever expecting to get a negative `itemId` passed here?

Reply:
Item variations are stored as negative values in config

Reply:
Probably you should just `mapToInt(Math::abs)` in BankTagsPlugin instead, since that seems like an odd detail to leave here